### PR TITLE
Added new dropdown button classes

### PR DESCRIPTION
**Pull request by czpython**

### CODE REVIEW

File: cms/templates/cms/toolbar/items/dropdown_button.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<a href="javascript: void 0" class="cms-btn cms-dropdown-toggle {% if active %} cms-btn-active{% endif %}{% if disabled %} cms-btn-disabled{% endif %} {{ extra_classes|join:' ' }}">
```
Comment (FinalAngel) [inline cms/templates/cms/toolbar/items/dropdown_button.html]: nope, #

File: cms/static/cms/sass/settings/_cms.scss [hunk 2]
```diff
@@ -330,6 +334,16 @@ $dropdown-item-icon-position: 14px;
 $dropdown-arrow-side: 10px;
 $dropdown-arrow-position: $submenu-right + $submenu-icon-area-size/2 - $dropdown-arrow-side/2;
 
+$dropdown-bg:           $submenu-dropdown-bgcolor;
```
Comment (FinalAngel) [inline cms/static/cms/sass/settings/_cms.scss]: would be nice to align

**Comment by czpython** [comment]
Looks good. Make sure to squash your commits though :)

**Comment by FinalAngel** [comment]
@czpython @vxsx the CHANGELOG also needs to be updated

**Comment by vxsx** [comment]
@czpython do you need to write any tests?

**Comment by vxsx** [comment]
force pushed prettier commit history

**Pull request merged**